<?xml version="1.0"?>
<!--Only use the left camera-->
<launch>
  <arg name="image_in" default="/vizzy/l_camera/image_raw" />
  <arg name="image_out" default="image_out" />
  <include file="$(find pedestrian_detector)/launch/starteyes.launch">
    <arg name="camera_left" value="true"/>
  </include>
  <node pkg="pedestrian_detector" type="detector" name="detector">
		<remap from="image_in" to="$(arg image_in)"/>
		<remap from="image_out" to="$(arg image_out)"/>
  </node>
</launch>
